common/static/common/js/components/views/paging.js

```diff
@@ -1,142 +0,0 @@
-define(["underscore", "backbone", "gettext", "common/js/components/views/paging_mixin"],
```

Comment:
As mentioned before, try to move this back rather than having it deleted and recreated.